### PR TITLE
feat: Update telemetry api endpoints

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Review by MiloradFilipovic — APPROVED** [review]
Looks good.

**Review by MiloradFilipovic — APPROVED** [review]
Re-approving after merge.

**Comment by janober** [comment]
Got released with `n8n@0.216.0`

**Pull request merged**
